src/lib/snyk-to-html.ts

```diff
@@ -93,7 +94,7 @@ function mergeData(dataArray: any[]): any {
   const totalDepCount =
     dataArray.reduce((acc, item) => acc + item.dependencyCount || 0, 0);
 
-  const paths = dataArray.map(d => d.path);
+  const paths = dataArray.map(d => ({path: d.path, packageManager: d.packageManager}));
```

Comment:
I know it was already here, but I'd vote for renaming `d` to something meaningfull - I don't even know, what it is :-)

Reply:
Thanks for reviewing, updated with a more sensible name.